.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,30 @@
+# Pull Request Template
+
+## Description
+
+Please include a summary of the change and which issue is fixed.
+
+Fixes # (youtrack issue)
+
+## Type of change
+
+Please delete options that are not relevant.
```

Comment:
I wouldn't delete any options. Just tich the applicable. The detail page of the PR is dynamic, so these are actual checkboxes, right?